.github/workflows/test.yml

```diff
@@ -3,22 +3,35 @@
 # These tests are currently written in TypeScript and are intended to be run in a Node.js environment.
 # This GitHub Action takes care of deploying to npm and GitHub.
 
-# TODO give every step a name
-
 name: Tests
 on:
     push:
         branches:
             - main
     pull_request: # Runs on pull requests to any branch
 env:
-    IS_RELEASE_CANDIDATE: ${{ contains(github.head_ref, 'release--') }}
-    IS_FEATURE: ${{ !contains(github.head_ref, 'release--') }}
-    DFX_VERSION: 0.21.0
+    DFX_VERSION: 0.22.0
     NODE_VERSION: 20
+    IS_MAIN_BRANCH: ${{ github.ref == 'refs/heads/main' }}
+    IS_RELEASE_BRANCH_PR: ${{ contains(github.head_ref, 'release--') }}
```

Comment:
Is there a startsWith instead of contains? Would be safer

Reply:
looks like there is